electron/constants.ts

```diff
@@ -30,6 +30,7 @@ const fontsStore = new Store({
 const isMac = platform() === 'darwin'
 const isWindows = platform() === 'win32'
 const isLinux = platform() === 'linux'
+const isRunningInSteam = 'SteamEnv' in env
```

Comment:
I wonder if this will cause Epic to open maximized/fullscreen even when not in Game Mode or Big Picture, and if it's really desired to open maximized/fullscreen in such situations.

Reply:
There is sadly no way to just detect Game Mode/Big Picture

Reply:
If someone wants to run heroic inside steam he is most likely in big picture or game mode. So this few people who don't want the fullscreen should open a Ticket.

Reply:
Fair enough.

Reply:
FYI:

> At least on SteamOS, when gamescope runs as a system session, you get `XDG_CURRENT_DESKTOP` set to `gamescope`.

https://github.com/Plagman/gamescope/issues/544#issuecomment-1178863974